### PR TITLE
Chapter 10

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: Ch10/Ch10.md [hunk 1]
```diff
@@ -528,21 +530,27 @@ Now, let's do some queries on this index.
...
+Consider the query:  
+`from index 'test/search' as n  
+where search(n.Search, "princess flower tower")`.
 
 If you run this query, you'll find that this actually matches the document, even though the word `flower` is nowhere to be found. That's because
 of the way RavenDB processes queries on full text search. It's enough that we have _any_ match to be considered enough to return the result. More 
 advanced options, such as phrase queries, are also available when using Lucene directly, such as the following queries:
 
-* `where lucene(n.Search, ' "princess tower" ')`&mdash;phrase query match (note the `"` in the query) because there's a `princess` followed by a `tower` in 
+* `where lucene(n.Search, "princess tower")` &mdash;
```
Comment (ayende) [inline Ch10/Ch10.md:542]: No, here, the ' "term " ` is intentional and really required

File: Ch10/Ch10.md [hunk 2]
```diff
@@ -528,21 +530,27 @@ Now, let's do some queries on this index.
...
 advanced options, such as phrase queries, are also available when using Lucene directly, such as the following queries:
 
-* `where lucene(n.Search, ' "princess tower" ')`&mdash;phrase query match (note the `"` in the query) because there's a `princess` followed by a `tower` in 
+* `where lucene(n.Search, "princess tower")` &mdash;  
+  Phrase query match (note the `"` in the query) because there's a `princess` followed by a `tower` in 
   the text, even though we don't have the `'` in the query.
-* `where lucene(n.Search, ' "running princess" ')`&mdash;also a match because the word `running` is followed by a `princess` (with the stop words removed).
-* `where lucene(n.Search, ' "running knight" ')`&mdash;not a match. There's no `running` followed by `knight` in the text.
+* `where lucene(n.Search, "running princess")` &mdash;  
+  Also a match because the word `running` is followed by a `princess` (with the stop words removed).
```
Comment (ayende) [inline Ch10/Ch10.md:546]: Same.

File: Ch10/Ch10.md [hunk 3]
```diff
@@ -528,21 +530,27 @@ Now, let's do some queries on this index.
...
+* `where lucene(n.Search, "running knight")` &mdash;  
+  Not a match. There's no `running` followed by `knight` in the text.
```
Comment (ayende) [inline Ch10/Ch10.md:548]: Same

File: Ch10/Ch10.md (hunk 1 above)
Comment (Danielle9897) [reply Ch10/Ch10.md:542]: done

File: Ch10/Ch10.md (hunk 2 above)
Comment (Danielle9897) [reply Ch10/Ch10.md:546]: done

File: Ch10/Ch10.md (hunk 3 above)
Comment (Danielle9897) [reply Ch10/Ch10.md:548]: done

**Pull request merged**